pipeline/eBOSS_QSO/QuasarMapper.py

```diff
@@ -0,0 +1,191 @@
+import matplotlib.pyplot as plt
+from astropy.io import fits
+from astropy.table import Table
+
+import os
+import pandas as pd
+import collections
+import numpy as np
+import healpy as hp
+
+import pymaster as nmt
+
+
+class QuasarMapper:
+    def __init__(self): 
+        #Inputs: the four fits files from eBOSS DR14
+        
+        data_path = '/home/zcapjru/PhD/Data/'
+        #Check for data files:
+        if os.path.exists(data_path+'eBOSS_QSO_clustering_data-NGC-vDR16.fits') :
+            print('NGC data found')
+            self.NGC_data =  fits.open(data_path + 'eBOSS_QSO_clustering_data-NGC-vDR16.fits')
+        else :
+            print('missing NGC data')
+
+        if os.path.exists(data_path+'eBOSS_QSO_clustering_data-SGC-vDR16.fits') :
+            print('SGC data found')
+            self.SGC_data =  fits.open(data_path + 'eBOSS_QSO_clustering_data-SGC-vDR16.fits')
+        else :
+            print('missing SGC data')
+           
+        if os.path.exists(data_path+'eBOSS_QSO_clustering_random-NGC-vDR16.fits') :
+            print('NGC random data found')
+            self.NGC_r_data =  fits.open(data_path + 'eBOSS_QSO_clustering_random-NGC-vDR16.fits')
+        else :
+            print('missing NGC random data')
+           
+        if os.path.exists(data_path+'eBOSS_QSO_clustering_random-SGC-vDR16.fits') :
+            print('SGC random data found')
+            self.SGC_r_data =  fits.open(data_path + 'eBOSS_QSO_clustering_random-SGC-vDR16.fits')
+        else :
+            print('missing SGC random data')
+        
+
+        self.nside = 512

```

Comment:
Make `nside` another input parameter (instead of hardcoding it). You can give it the default value of 512 if you want.